### DISCUSSION TITLE
✅ ESC Key on external keyboard

### DISCUSSION THREAD

**Discussion by BobDenny** [isontheline/pro.webssh.net#635] [General]
I use vim a lot, and I have an external keyboard. Is there any way to prevent ESC (heavily used in vim) from closing the app back to its icon? Poof! I have to use a 2-key combination Fn+Esc.

**Comment by isontheline** [comment]
Firstly thank you so much @BobDenny for your support about my work and your sponsorship 🙏 ❤️

What's your keyboard brand?

Yes you're right on some keyboard some combo is needed in order to do "esc".

In case your keyboard doesn't have an ESC key you can try this custom WebSSH combo : https://webssh.net/documentation/help/SSH/keyboard-shortcuts-combo-keys/

**Reply by BobDenny** [reply]
My apologies. This is an Arteck BT keyboard. I looked more carefully and the Esc key is pale blue for ESC and that means you need to hold the pale blue Fn key for ESC. Duh. Without Fn is it a box, meaning "home" or "Close app".  I discovered that my  WebSSH was still V7 also, it didn't auto-update. Now I have  V8 and it's even better!! Thanks! This is really a beautiful app. My SSH connections are Public Key and it all works perfectly.

**Reply by isontheline** [reply]
Thank you so much Bob 🙏
